src/plugins/watcher.ts

```diff
@@ -0,0 +1,65 @@
+import { walletData } from "@/plugins/walletData";
+import { iWallet } from "@/types/lib";
+import { Dispatch } from "vuex";
+import { Store } from "vuex/types/index";
+
+import { ethWindow } from "~/plugins/build";
+
+let changeNetworkWasSet = false;
+
+/**
+ * @todo avoid cross-colling
+ * @param dispatch
+ * @param context
+ * @return {function(): Promise<void>}
+ */
+const changeAccountHandle = (dispatch: Dispatch, context: Store<iWallet>) => {
+  context.app.$toast.global.zkException({
+    message: "You've changes active account. Restarting the DAPP",
```

Comment:
```suggestion
    message: "You've changes active account. Restarting the dApp",
```
It is more often used as dApp instead of DAPP (just like App for mobile)